docs/whats-new.asciidoc

```diff
@@ -4,137 +4,127 @@
 {security-guide-all}/7.9/whats-new.html[7.9]
 
 // NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
 // tag::notable-highlights[]
 
 [float]
-== Detection rules and alerts enhancements
+== Generative AI enhancements
 
-The following enhancements have been added to detection rules and alerts:
+The following generative AI enhancements have been added:
 
 [float]
-=== Per-field diff for Elastic prebuilt rule updates
+=== Attack Discovery
 
-When examining an {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[updated Elastic prebuilt detection rule], you can now view rule changes field by field as well as in a full JSON view.
+Introducing {security-guide}/attack-discovery.html[Attack discovery], a new AI-powered tool that can identify and describe potential attacks that span multiple alerts. It can identify connections between alerts and map them to the MITRE ATT&CK® matrix to help make the most of each security analyst's time, fight alert fatigue, and reduce your mean time to respond.
 
 [role="screenshot"]
-image::whats-new/images/8.13/prebuilt-rules-update-diff.png[Prebuilt rule comparison, 85%]
+image::whats-new/images/8.14/attack-discovery-full-card.png[Attack discovery detail view]
 
 [float]
-=== Alert suppression supported for indicator match rules
+=== Elastic AI Assistant UI improvements
 
-{security-guide}/alert-suppression.html[Alert suppression] now supports the {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match] rule type. You can use it to reduce the number of repeated or duplicate detection alerts created by an indicator match rule.
+{security-guide}/security-assistant.html[Elastic AI Assistant] for security has a redesigned user interface that uses a flyout instead of a popup, making it fit more smoothly within your workflows and with standard {kib} design patterns. Also, when using OpenAI models, responses from AI Assistant can now "stream", meaning render word-by-word rather than appearing as complete text blocks after processing is complete, which provides a more conversational experience.
```

Comment:
```suggestion
{security-guide}/security-assistant.html[Elastic AI Assistant] for {elastic-sec} has a redesigned user interface that uses a flyout instead of a popup, making it fit more smoothly within your workflows and with standard {kib} design patterns. Also, when using OpenAI models, AI Assistant can now "stream" responses, rendering word-by-word rather than appearing as complete text blocks, to provide a more conversational experience.
```